classes/form/CustomerForm.php

```diff
@@ -146,13 +147,25 @@ public function validate()
         }
 
         $passwordField = $this->getField('password');
-        if ((!empty($passwordField->getValue()) || $this->passwordRequired)
-            && Validate::isPlaintextPassword($passwordField->getValue()) === false) {
-            $passwordField->addError($this->translator->trans(
-                'Password must be between 5 and 72 characters long',
-                [],
-                'Shop.Notifications.Error'
-            ));
+        if (!empty($passwordField->getValue()) || $this->passwordRequired) {
+            if (Validate::isAcceptablePasswordLength($passwordField->getValue()) === false) {
+                $passwordField->addError($this->translator->trans(
+                    'Password must be between %d and %d characters long',
```

Comment:
Why not %s?

Reply:
@Julievrz Because `%d` is for double/integers in C (`sprintf` comes from the C world).

Reply:
`%s` is for strings

Reply:
Thanks for explaining :-)